product_docs/docs/epas/14/epas_inst_linux/managing_an_advanced_server_installation.mdx

```diff
@@ -285,7 +285,7 @@ In addition to the cluster configuration options documented in the PostgreSQL co
 
 `--no-redwood-compat`
 
-Include the `--no-redwood-compat` keywords to instruct the server to create the cluster in PostgreSQL mode. When the cluster is created in PostgreSQL mode, the name of the database superuser is `postgres`, the name of the default database is `postgres`, and EDB Postgres Advanced Server’s features compatible with Oracle databases aren't available to the cluster.
+Include the `--no-redwood-compat` keywords to instruct the server to create the cluster in PostgreSQL mode. When the cluster is created in PostgreSQL mode, the name of the database superuser is `postgres`, the name of the default database is `postgres`. The few Advanced Server’s features compatible with Oracle databases will be available with this mode. However, we recommend using the Advanced server in redwood compatibility mode to use all its features.
```

Comment:
might be worth checking with Betsy, but it seems to me like there should be an "and" before "the name of the default database is `postgres`."

Reply:
Does this sound better in stead of the last two lines?

"Not all Oracle-compatible features in Advanced Server are available with this mode. We recommend using the Advanced server in redwood compatibility mode to use all the Oracle-compatible features."

Reply:
I have added "and"